pkg/operator/ceph/cluster/osd/topology/topology.go

```diff
@@ -56,66 +57,67 @@ func ExtractOSDTopologyFromLabels(labels map[string]string) (map[string]string,
 	return topology, topologyAffinity
 }
 
+func rookTopologyLabelsOrdered() []string {
+	topologyLabelsOrdered := []string{}
+	for i := len(CRUSHTopologyLabels) - 1; i >= 0; i-- {
+		label := CRUSHTopologyLabels[i]
+		topologyLabelsOrdered = append(topologyLabelsOrdered, topologyLabelPrefix+label)
+	}
+	return topologyLabelsOrdered
+}
+
+func allKubernetesTopologyLabelsOrdered() []string {
+	return append([]string{corev1.LabelTopologyRegion,
+		corev1.LabelTopologyZone,
+		labelHostname},
```

Comment:
hostname should be after all the rookTopologyLabelsOrdered. The host is the lowest level in the crush map.

Reply:
Moved